tests/lib/snaps/test-snapd-with-configure-core18/meta/snap.yaml

```diff
@@ -3,3 +3,8 @@ version: 1.0
 summary: Basic snap with a configure hook for core18
 description: A basic snap with a passthrough configure hook for core18
 base: core18
+
+apps:
+  some:
```

Comment:
wondering if adding a service to these is confusing or not, not a blocker

Reply:
Yeah, I'd also prefer a dedicated snap. The problem with changing any of the shared snaps is that one needs to evaluate how it impacts all the tests using it.